src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zoo {
```

Comment:
let's divide logic of storing animal (Zoo) and main method (creating animals, starting program)

Reply:
done